### DISCUSSION TITLE
[platform/gitea] Sort autodiscover by updated

### DISCUSSION THREAD

**Discussion by viceice** [renovatebot/renovate#9154] [Suggest an Idea]
I'm thinking if it makes sense to let order gitea the autodiscover repos by last updated. So recent updated repos would be processed first. By default it will be sorted by repo name (without org).

![image](https://user-images.githubusercontent.com/1798109/111277439-bea97800-8638-11eb-985e-e38d3debc0b0.png)

Any Ideas / remarks / suggestions / preferences to this behavior change?

[the remark this answers is not in the document]
**Reply by rarkins** [reply]
I recall someone else asking about this, maybe for GitLab. But if the whole queue gets processed eventually anyway, what's the main thing it achieves?

[the remark this answers is not in the document]
**Reply by viceice** [reply]
I'm not really sure about the benefit, i found the sort filter and thought it might me a good idea.

So i want to discuss it first it if really makes sense to process recently updated repos before older ones. I think this highly depends on overall processing time.

#9153 will have a much bigger benefit.
